### ISSUE TITLE
[DMP Review] Provide a option to activate / deactivate the review

### ISSUE THREAD

**Issue by MyPyDavid** [rdmorganiser/rdmo#973]
### Rationale / Begründung

As is described in #972 , this Review Service should be able to be activated or deactivated by a User in the project detail page.

### Affected

Users

### Minimal functionality

* Provide a option to activate / deactivate the review

### Nice-to-have functionality

*

### References / Verweise

* https://github.com/search?q=repo%3Ardmorganiser%2Frdmo+%5BDMP+Review%5D&type=issues

**Comment by jochenklar** [comment]
Currently we have no means to store user input outside of a project. (Except maybe the state of the overlays.) This might be more complicated then expected.

**Comment by MyPyDavid** [comment]
Alternatively, it could be stored in a `Value` on a certain `Attribute`, like in the zenodo plugin that stores the pid for the project. It only requires a certain 'hard-coded` attribute.

Could we not claim 🎏  a terms namespace for this, eg. `https://rdmorganiser.github.io/terms/project/metadata/` ??

**Comment by jochenklar** [comment]
Yes, but we should decide if we extend the user model for things like this, or the domain, not both.

**Comment by jochenklar** [comment]
Summarizing the discussion with @MyPyDavid today: we tend towards the "domain solution":
* We can use a list `PROJECT_METADATA` to determine attributes which are used as project Metadata and can be changed (or not) in the update/create project interface.
* We could extend `Attribute` with a `meta` boolean, which prevents managers from using this attribute in questions (which would be beneficial for plugins as well).
